conan/cli/commands/compare.py

```diff
@@ -0,0 +1,233 @@
+import json
+import subprocess
+import os
+
+from conan.errors import ConanException
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput
+from conan.cli.command import conan_command
+from conan.internal.util.files import chdir
```

Comment:
The idea is that ``cli.commands`` layer doesn't use anything from ``conan.internal``, only from public and mainly from api

Reply:
Yes, this is in fact an unused import leftover from a previous implementation 👍